includes/course-theme/class-sensei-course-theme.php

```diff
@@ -84,7 +84,6 @@ public function init() {
 		add_action( 'registered_post_type', [ $this, 'add_post_type_rewrite_rules' ], 10, 2 );
 		add_action( 'registered_taxonomy', [ $this, 'add_taxonomy_rewrite_rules' ], 10, 3 );
 		add_action( 'shutdown', [ $this, 'maybe_flush_rewrite_rules' ] );
-		add_action( 'setup_theme', [ $this, 'maybe_override_theme' ], 2 );
```

Comment:
We should also remove the `maybe_override_theme` method. It's not used anywhere else.

Reply:
I think we still need this too if we keep the filter. When they set it to true, we'll need to redirect to `/learn` and do the override here as before.